diff --git a/node_modules/@imtbl/checkout-widgets/dist/browser/CommerceWidget-0qeuTFA8.js b/node_modules/@imtbl/checkout-widgets/dist/browser/CommerceWidget-0qeuTFA8.js
index da3ed57..09c68bd 100644
--- a/node_modules/@imtbl/checkout-widgets/dist/browser/CommerceWidget-0qeuTFA8.js
+++ b/node_modules/@imtbl/checkout-widgets/dist/browser/CommerceWidget-0qeuTFA8.js
@@ -341,6 +341,12 @@ function mapPurchaseWidgetEvent(event) {
                 type: fo.CLOSE,
                 data: {},
             };
+        //-------------------------------------------------------------------------------------
+        case "checkout-start":  // PeterG - I added the below to handle unknown event type 16/4/25
+            return "CHECKOUT_START";
+        case "connect-success":
+            return "CONNECT_SUCCESS"; // or return null / log only
+         //-------------------------------------------------------------------------------------
         default:
             throw new Error(`Unknown purchase event type "${event.detail.type}"`);
     }
